### PR TITLE
create README.md in 09.gan

### PR DESCRIPTION
**Pull request by acosta123**
add the English version README.md

### CODE REVIEW

File: 09.gan/README.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
+# GAN(Generative Adversarial Networks)
+
+The source codes of this tutorial are in book/09.gan . Please refer to the instructions to Book document for first use.
+
+## Backgrounds
+
+GAN \[[1](#References)\] is a kind of unsupervised learning method, which learns through games between two neural networks. This method was proposed by lan·Goodfellow et al in 2014, for whose paper you can refer to [Generative Adversarial Network](https://arxiv.org/abs/1406.2661)。
+
+GAN is constituted by a generative network and a discrimination network. The generative network takes random sampling from latent space as input, while its output results need to imitate the real samples in training set to the greatest extent. The discrimination network takes real samples or the output of the generative network as input, aimed to distinguish the output of the generative network from real samples. And the generative network tries to cheat the discrimination network. These two networks confront each other and adjust parameters constantly in order to tell the samples generated by the generative network and the real samples apart. \[[2](#References)\] ）.
+
```
Comment (tink2123) [inline 09.gan/README.md]: remove the ")", same as below
```suggestion
\[[2](#References)\] .
```

File: 09.gan/README.md [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+GAN is commonly used to generate convincing pictures that can \[[3](#References)\] ）. What's more, it can also generate videos and 3D object models etc.
```
Comment (tink2123) [inline 09.gan/README.md]: that can what?

File: 09.gan/README.md [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+
+## Effect Display
+
+In this tutorial, MNIST data set are input to the network for training. After training for 19 turns, we can see that the generative pictures are very close to the real pictures. In the figure below, the first eight rows show real pictures and the rest show pictures generated by the network:
```
Comment (tink2123) [inline 09.gan/README.md]: 19 turns ==> 19 epochs

File: 09.gan/README.md [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+
+<p align="center">
+<img src="https://raw.githubusercontent.com/PaddlePaddle/book/develop/09.gan/image/dcgan_g.png" width="550"><br/>
+figure 3. Generator(G) in DCGAN
+</p>
+
+
+## Data Preparation
+
+In this tutorial, MNIST of small data size is used to train Generator ande Discriminator , which can be downloaded to lacal automatically by paddle.dataset module.
```
Comment (tink2123) [inline 09.gan/README.md]: 这一段优化一下表述？
另外 ande ==》 and，   lacal ==》 local

File: 09.gan/README.md [hunk 5]
```diff
@@ -0,0 +1,438 @@
...
+
+Please refer to [Digit Recognition](https://github.com/PaddlePaddle/book/tree/develop/02.recognize_digits) for specific introduction to NMIST.
+
+## Train the Model
+
+ `09.gan/dc_gan.py` demonstrates the whole training process.
+
+### Load the Package
+
+First load the Fluid and other relaterd packages of PaddlePaddle.
```
Comment (tink2123) [inline 09.gan/README.md]: relaterd ==》 related

File: 09.gan/README.md [hunk 6]
```diff
@@ -0,0 +1,438 @@
...
+        num_filters=num_filters,
+        pool_size=2,
+        pool_stride=2,
+        param_attr=name + 'w',
+        bias_attr=name + 'b',
+        use_cudnn=use_cudnn,
+        act=act)
+```
+
+- Fully Connected Layer
```
Comment (tink2123) [inline 09.gan/README.md]: Fully Connected ==》 Full connect

File: 09.gan/README.md [hunk 7]
```diff
@@ -0,0 +1,438 @@
...
+        bias_attr=name + '2',
+        moving_mean_name=name + '3',
+        moving_variance_name=name + '4',
+        name=name,
+        act=act)
+```
+
+- convolution layer
+
+Call `fluid.nets.simple_img_conv_pool` to realize pooling of convolution. The kernel dimension is 3x3, the pooling window dimension is 2x2, the window slide step size is 2, and the activation function is appointed by certain network structure.
```
Comment (tink2123) [inline 09.gan/README.md]: kernel == > filter size 
pooling window dimension == > pooling size
window slide step size == > pool stride size

File: 09.gan/README.md [hunk 8]
```diff
@@ -0,0 +1,438 @@
...
+        stride=stride,
+        dilation=dilation,
+        padding=padding,
+        use_cudnn=use_cudnn,
+        act=act)
+```
+
+- Discriminator
+
+Discriminator uses real data set and fake pictures generated by Generator to train at the same time, and try to make the output result 1 in the real data set case and make it 0 in the fake case. In this tutorial, Discriminator realized is constituted by two convolution pooling layers and two fully connected layers, in which the neuron number of the lat FCL is 1, outputting a dichotomy result.
```
Comment (tink2123) [inline 09.gan/README.md]: lat 是什么？
FCL ==》 FC
dichotomy ==》 Binary classification
这段建议再优化一下

File: 09.gan/README.md [hunk 9]
```diff
@@ -0,0 +1,438 @@
...
+Take adam as optimizer to optimize the loss of judging real images and generative images respectively.
+
+```python
+opt = fluid.optimizer.Adam(learning_rate=LEARNING_RATE)
+opt.minimize(loss=d_loss)
+parameters = [p.name for p in g_program.global_block().all_parameters()]
+opt.minimize(loss=dg_loss, parameter_list=parameters)
+```
+
+### Data Set Feeders Configuration
```
Comment (tink2123) [inline 09.gan/README.md]: data set ==》 dataset 
请将全文中的相应部分 一起修改

File: 09.gan/README.md [hunk 10]
```diff
@@ -0,0 +1,438 @@
...
+            print(msg)
+            plt.title(msg)
+            plt.savefig(
+                '{}/{:04d}_{:04d}.png'.format(output, pass_id,
+                                              batch_id),
+                bbox_inches='tight')
+            plt.close(fig)
+```
+
+print generative results in a certain turn:
```
Comment (tink2123) [inline 09.gan/README.md]: turn ==》 epoch

**Review by tink2123 — COMMENTED** [review]
good～some comments for reference
